drake/systems/@DynamicalSystem/DynamicalSystem.m

```diff
@@ -719,5 +719,8 @@
   
   properties (Access=public)
     warning_manager;
-  end    
+  end
+  properties (SetAccess=protected)
+    initial_state = [];
```

Comment:
i don't think we should have this...?  we already have the getInitialState method.